elixir/__init__.py

```diff
@@ -84,3 +84,14 @@ def model_size_formatter(numel: int) -> str:
         return f'{numel / KB_SIZE:.1f}K'
     else:
         return str(numel)
+
+
+def calc_buffer_size(m: nn.Module, test_dtype: torch.dtype = torch.float):
```

Comment:
don't use a single character `m` for the argument name.

Reply:
Meanwhile, this looks like a util function, it should not be put in `__init__` and remember to add docstring whenever appropriate to help others understand your code logic.